bisection.py

```diff
@@ -181,7 +181,7 @@ def get_mid_commit(self, left: Commit, right: Commit) -> Optional[Commit]:
     def setup_build_env(self, env) -> Dict[str, str]:
         env["USE_CUDA"] = "1"
         env["BUILD_CAFFE2_OPS"] = "0"
-        env["USE_XNNPACK"] = "0"
+        env["USE_XNNPACK"] = "1" # Customized for LTC.
```

Comment:
it's really confusing 
1) we need XNNPACK to make torchbench happy
   (even though we don't understand why XNNPack is being used, since we're benchmarking cuda)
2) XNNPACK is disabled for normal torchbench CI

???